### PR TITLE
chore(ci): add a mergify batch queue for external PRs

### PR DESCRIPTION
**Pull request by jxs**
and remove the if conditions no longer required as we are running them again on every PR

### CODE REVIEW

File: .github/workflows/interop-test.yml [hunk 1]
```diff
@@ -56,4 +56,5 @@ jobs:
           s3-cache-bucket: ${{ vars.S3_LIBP2P_BUILD_CACHE_BUCKET_NAME }}
           s3-access-key-id: ${{ vars.S3_LIBP2P_BUILD_CACHE_AWS_ACCESS_KEY_ID }}
           s3-secret-access-key: ${{ secrets.S3_LIBP2P_BUILD_CACHE_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ap-southeast-2
```
Comment (guillaumemichel) [inline .github/workflows/interop-test.yml]:
```suggestion
          aws-region: ${{ vars.S3_LIBP2P_BUILD_CACHE_AWS_REGION }}
```

Comment (jxs) [reply .github/workflows/interop-test.yml]: Thanks Gui!

**Review by guillaumemichel — APPROVED** [review]
Seems like a great fix! Maybe we need to document somewhere to use the `send-it-batch` tag instead of `send-it`?

**Comment by jxs** [comment]
Changed this PR, 
basically what is happening is that the secrets and variables used for the S3 access on the `interop-tests` are not exposed to PRs from external forks, see [here](https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/) for more info.

The solution that I found is using Mergify's [batch_size](https://docs.mergify.com/merge-queue/batches/) for PRs from external forks. What it does is creating another PR from within the organization that runs the whole CI suite (including the `interop-tests`). If the CI passes it merges the original PR.
To add to this new queue instead of using the `send-it` label, we use the `send-it-batch` label.
CC @guillaumemichel @dariusc93 @elenaf9

**Comment by jxs** [comment]
> Seems like a great fix! Maybe we need to document somewhere to use the `send-it-batch` tag instead of `send-it`?

yup makes sense Gui! I added an explanation the label description:
image
also added a comment to the `send-it-batch` on the `mergify.yml` config

**Review by dariusc93 — APPROVED** [review]

**Pull request merged**